ovirt-imageio.spec.in

```diff
@@ -56,7 +56,7 @@ Transfer disk images on oVirt system.
 %define __python3 /usr/bin/python3
 %endif
 
-%py3_build
+%py3_build "--executable=%{quote:/usr/bin/python3 -s}"
```

Comment:
Can we used this method to replace the all the ugly hacks? Call once with the system python, and if needed, call again with the special python (e.g. 3.9).